src/CallHandler.js

```diff
@@ -97,19 +99,54 @@ function pause(audioId) {
     }
 }
 
+function _reAttemptCall(call) {
+    if (call.direction === 'outbound') {
+        dis.dispatch({
+            action: 'place_call',
+            room_id: call.roomId,
+            type: call.type,
+        });
+    } else {
+        call.answer();
+    }
+}
+
 function _setCallListeners(call) {
     call.on("error", function(err) {
         console.error("Call error: %s", err);
         console.error(err.stack);
-        call.hangup();
```

Comment:
do we not still need this?

Reply:
This is handled within the js-sdk now - in general we should not be hanging up the call on errors since not all errors are fatal , eg. we may have failed to accept the call but that doesn't mean we (or another client) can't try again.